components/service/nimbus/src/main/java/mozilla/components/service/nimbus/ui/NimbusExperimentItemViewHolder.kt

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.nimbus.ui
+
+import android.view.View
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import mozilla.components.service.nimbus.R
+import org.mozilla.experiments.nimbus.EnrolledExperiment
+
+class NimbusExperimentItemViewHolder(
```

Comment:
As this is not public face class could be marked as internal

Reply:
I think this errors on Fenix because we are exposing some internal publicly through NimbusExperimentAdapter.